.github/workflows/java.yaml

```diff
@@ -7,7 +7,8 @@ jobs:
     steps:
       - uses: actions/checkout@master
       - name: Setup java
-        uses: actions/setup-java@v1
+        uses: actions/setup-java@v3
         with:
           java-version: 13
-      - run: mvn compile test
\ No newline at end of file
+          distribution: 'zulu'
```

Comment:
Zulu?

Reply:
On setup-java@v3 you need to specify the distribution, Zulu was the one @JesperTerkelsen suggested 🙂